src/app/store.ts

```diff
@@ -11,14 +11,47 @@ import {
   UrlInputState,
 } from "../features/urlInput";
 import { TabsActions, tabsReducer, TabsState } from "../features/tabs";
-
 export type ApplicationState = {
   knownTags: KnownTagsState;
   items: ItemsState;
   urlInput: UrlInputState;
   tabs: TabsState;
 };
-export default configureStore<
+
+const localStorages = ["knownTags", "items", "tabs"];
+
+function saveToLocalStorage(state: ApplicationState) {
+  try {
+    localStorages.map((storage) => {
+      const storePart = storage as keyof typeof state;
+      const serialisedState = JSON.stringify(state[storePart]);
+      localStorage.setItem("LinkBiscuit_" + storage, serialisedState);
```

Comment:
Maybe I'd consider extracting the `LinkBiscuit_` string to a top level `const` so that it is a bit more explicit. Something like `const linkBiscuitPrefix = 'LinkBiscuit'`. Also you could leverage string interpolation, e.g. `...setItem(`${linkBiscuitPrefix}_${storage}`).

Reply:
Smart!